### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

Release 1.9.0

### CODE REVIEW

**Review by ronijaakkola — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- An example of asking the user’s gender in a form to the Form pattern documentation
+
+#### Changed
+
+- [Koros] Added an example of a rotated wave motif shape
+- [SearchInput] Added an placeholder text to all Playground and code examples
+
+#### Fixed
+
+- Resolved some critical dependency security threats for new documentation site
```
Comment (minevala) [inline CHANGELOG.md:49]: for new documentation -> for **the** new documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+#### Added
+
+- WP and Drupal notes to the developers’ Getting Started documentation
+- An example of asking the user’s gender in a form to the Form pattern documentation
+
+#### Changed
+
+- [Koros] Added an example of a rotated wave motif shape
+- [SearchInput] Added an placeholder text to all Playground and code examples
```
Comment (minevala) [inline CHANGELOG.md:45]: an placeholder  -> a placeholder

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+#### Changed
+
+- Updated HDS Sketch libraries to Sketch 81 (81.1)
+
+### Documentation
+
+#### Added
+
+- WP and Drupal notes to the developers’ Getting Started documentation
+- An example of asking the user’s gender in a form to the Form pattern documentation
```
Comment (minevala) [inline CHANGELOG.md:40]: in a form to -> in a form **in**

**Review by minevala — COMMENTED** [review]
Nice release

File: CHANGELOG.md (hunk 3 above)
Comment (minevala) [reply CHANGELOG.md:40]: The current version is also ok since it is connected to the "added" word.

File: CHANGELOG.md (hunk 2 above)
Comment (VilleMiekkoja) [reply CHANGELOG.md]: - [ ] Improve grammar

File: CHANGELOG.md (hunk 3 above)
Comment (VilleMiekkoja) [reply CHANGELOG.md]: - [ ] Improve grammar

File: CHANGELOG.md (hunk 1 above)
Comment (VilleMiekkoja) [reply CHANGELOG.md]: - [ ] Improve grammar

**Review by minevala — APPROVED** [review]
It is good to go 👍 🎉

**Pull request merged**